docs/getting-started-provider-dev.md

```diff
@@ -148,7 +141,7 @@ and versioning information.
 
 Following the [GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) guidelines for Kubernetes development, set up your environment and get the latest development repository. Begin by forking both the Kubernetes and Cloud-Provider-OpenStack projects into your GitHub into your local workspace (or bringing your current fork up to date with the current state of both repositories).
 
-`make` will build, test, and package this project. This project uses [go dep](https://golang.github.io/dep/) for dependency management.
+`make` will build, test, and package this project. This project uses [go modules](https://github.com/golang/go/wiki/Modules) for dependency management since 1.13.
```

Comment:
go mod is supported since v1.17 fo cloud-provider-openstack

Reply:
yeah:

commit 80d14660d6d7756a7d3f2f832c76cc3e3ab8d583
Author: kayrus <kayrus@users.noreply.github.com>
Date:   Tue Nov 5 08:28:38 2019 +0100

    Switch dependency manager to go modules (#762)

    * Switch dependency manager to go modules

    * Update k8s to 1.6.2

    * Don't modify the current package dependencies on go tools installation

    * Update k8s to v1.17.0-alpha.3

    * Bump gophercloud

Reply:
Sure. got it.